.gitmodules

```diff
@@ -1,40 +1,32 @@
 [submodule "packages/contracts/lib/Uniswap/v2-periphery"]
 	path = packages/contracts/lib/Uniswap/v2-periphery
 	url = https://github.com/Uniswap/v2-periphery.git
-	tag = v1.0.0-beta.0
+	branch = master
```

Comment:
I guess `tag` option would be more stable because there would be breaking changes in the `master` branch but if we followed up on them asap, it will be broken for a while

Reply:
Can you check this comment @rndquu ?

Reply:
@0xcodercrane only [branch names are supported](https://stackoverflow.com/questions/1777854/how-can-i-specify-a-branch-tag-when-adding-a-git-submodule) so it's ok to remove the `tag` and use the `branch` instead. Besides some of the [libs](https://github.com/vittominacori/solidity-linked-list/branches) don't have branches for specific versions so we have to use the `master` branch

Reply:
interesting. will follow up on the current one